src/Button/index.js

```diff
@@ -2,60 +2,112 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-import { Container, ContainerIconLeft, ContainerIconRight } from './elements';
+import { Container, ContainerIcon } from './elements';
 
-const displayIconLeft = (side, icon) => {
-  return side === 'right' || icon === null ? null : <ContainerIconLeft>{icon}</ContainerIconLeft>;
-};
-const displayIconRight = (side, icon) => {
-  return side === 'left' || icon === null ? null : <ContainerIconRight>{icon}</ContainerIconRight>;
-};
+/**
+ * Button
+ *
+ * This component is in charge of displaying
+ * a button
+ *
+ * @param {string} appearance // The base styling to apply to the button.
+ * @param {node} children // Anything that can be rendered: numbers, strings, elements or an array (or fragment).
+ * @param {string} className // ClassName needed by styled components.
+ * @param {bool} fluid // Whether the button is fluid or not.
+ * @param {bool} disabled // Whether the button is disabled or not.
+ * @param {node} icon // Add icon node here to illustrate the text aside.
+ * @param {enum} icon // Set the icon position compared to the text. It will be set to the left side by default.
```

Comment:
`@param {enum}` doesn't exists in JSDOC. 
See : http://usejsdoc.org/tags-enum.html